src/SqlPersistence/Config/SqlDialect_MsSqlServer.cs

```diff
@@ -52,6 +53,12 @@ internal override CommandWrapper CreateCommand(DbConnection connection)
                 return new CommandWrapper(command, this);
             }
 
+            internal override void AddExtraDiagnosticsInfo(Dictionary<string, object> diagnostics)
+            {
+                diagnostics.Add("CustomSchema", string.IsNullOrEmpty(Schema));
```

Comment:
Can we add the actual schema name here?

Reply:
I don't think so. The name of the schema is not a security sensitive info. Connection strings are a different story because you can embed credentials in them.